ctapipe/containers.py

```diff
@@ -1044,3 +1088,86 @@ class ArrayEventContainer(Container):
         "Container for calibration coefficients for the current event",
     )
     mon = Field(MonitoringContainer(), "container for event-wise monitoring data (MON)")
+
+
+class SchedulingBlockContainer(Container):
+    """Stores information about the scheduling block. This is a simplified version
+    of the SB model, only storing what is necessary for analysis."""
+
+    container_prefix = "sb"
+    id = Field(-1, "Scheduling block ID", type=np.int64)
+    type = Field(
+        SchedulingBlockType.OBSERVATION,
+        description="Type of scheduling block",
+        type=SchedulingBlockType,
+    )
+    observing_mode = Field(ObservingMode.CUSTOM, "Observing mode", type=ObservingMode)
+    pointing_mode = Field(
+        None,
+        "Pointing Mode is the coordinate frame in which the pointing position is non-moving",
+        type=CoordinateFrameType,
+    )
+
+
+class ObservationBlockContainer(Container):
+    """Stores information about the observation"""
+
+    container_prefix = "obs"
+    id = Field(None, "Observation Block ID", type=np.int64)
```

Comment:
We call this `obs_id` every where else. This is a bit of a question of naming conventions for foreign key / primary keys.

I normally also use `id` in the actual table and then `<name>_id` when used somewhere else as foreign key, so this fits, but something to be kept in mind.